libs/wingsdk/src/std/map.ts

```diff
@@ -39,10 +39,11 @@ export class Map {
    * If the value that is associated to the provided key is an object, then you will get a reference
    * to that object and any change made to that object will effectively modify it inside the map.
    *
-   * @macro ($self$)[$args$]
+   * @macro ((map, key) => { return map[key] ?? throw new Error("no value found for key '" + key + "'"); })($self$, $args$)
```

Comment:
Check should be more explicit to handle null (nil) values

```suggestion
   * @macro ((map, key) => { return map[key] === undefined ?? throw new Error("no value found for key '" + key + "'"); })($self$, $args$)
```

Reply:
maybe

```suggestion
   * @macro ((map, key) => key in map ? map[key] : throw new Error("no value found for key '" + key + "'"))($self$, $args$)
```

Reply:
You are right. Let's do it with the `in` operator, I agree. Makes the code more explicit too